ui/src/tasks/actions/index.ts

```diff
@@ -296,7 +321,7 @@ export const updateTaskStatus = (task: Task) => async dispatch => {
 
 export const updateTaskName = (task: Task) => async dispatch => {
   try {
-    await client.tasks.update(task.id, task)
+    await patchTask({taskID: task.id, data: task})
```

Comment:
Since this only updates the name (and does not pass along any information relating the `cron`, `every` specifics), I did not run a check to see whether or not the cron / every props needed to be updated